app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationViewModel.kt

```diff
@@ -187,7 +187,8 @@ class ConversationViewModel @Inject constructor(
         conversationViewState = conversationViewState.copy(
             conversationName = conversationName,
             conversationAvatar = conversationAvatar,
-            conversationDetailsData = conversationDetailsData
+            conversationDetailsData = conversationDetailsData,
+            isConversationMember = conversationDetails.conversation.removedBy == null
```

Comment:
If the user leaves the conversation on their own, does this work as well?

Reply:
This `removedBy` field will be deleted once https://github.com/wireapp/kalium/pull/786 has been merged

Reply:
Instead you should use `conversationRepository.isUserMember()` to know whether the selfUser is still member of that conversation or not.